kani-compiler/src/codegen_cprover_gotoc/codegen/typ.rs

```diff
@@ -891,11 +899,12 @@ impl<'tcx> GotocCtx<'tcx> {
             // https://github.com/model-checking/kani/issues/214
             ty::FnPtr(_) => self.codegen_ty(pointee_type).to_pointer(),
 
+            ty::Generator(_, _, _) => self.codegen_ty(pointee_type).to_pointer(),
+
             // These types have no regression tests for them.
             // For soundness, hold off on generating them till we have test-cases.
             ty::Bound(_, _) => todo!("{:?} {:?}", pointee_type, pointee_type.kind()),
             ty::Error(_) => todo!("{:?} {:?}", pointee_type, pointee_type.kind()),
-            ty::Generator(_, _, _) => todo!("{:?} {:?}", pointee_type, pointee_type.kind()),
             ty::GeneratorWitness(_) => todo!("{:?} {:?}", pointee_type, pointee_type.kind()),
```

Comment:
Can we do the same for `GeneratorWitness`?

Reply:
I would rather wait till we see a testcase first. The documentation for `GeneratorWitness` https://doc.rust-lang.org/stable/nightly-rustc/rustc_middle/ty/relate/struct.GeneratorWitness.html doesn't have an explanation of when it would arise.